### ISSUE TITLE
toText missing BYSETPOS

### ISSUE THREAD

**Issue by gbulfon** [jkbrzt/rrule#629]
Hello, I think I found a bug.

RRule.fromString("FREQ=MONTHLY;BYSETPOS=3;BYDAY=TH;INTERVAL=3;UNTIL=20240730T220000Z").toText();

returns : 'every 3 months on Thursday until July 30, 2024'
should be : 'every 3 months on 3rd Thursday until July 30, 2024'

Is the BYSETPOS check missing?

**Comment by Corepex** [comment]
It seems that the text version of `BYSETPOS` is missing in every configuration

**Comment by Corepex** [comment]
This should be the solution: https://github.com/jkbrzt/rrule/pull/504

**Comment by gbulfon** [comment]
> This should be the solution: #504

Yes I found it but it's not been incorporated. I had to patch it myself. Why?
